src/main/telemetry/hott.c

```diff
@@ -509,8 +456,28 @@ static void hottCheckSerialData(uint32_t currentMicros)
 }
 
 static void hottSendTelemetryData(void) {
+    if (!hottIsSending) {
+        hottIsSending = true;
+        // FIXME temorary workaround for HoTT not working on Hardware serial ports due to hardware/softserial serial port initialisation differences
+        if ((portConfig->identifier == SERIAL_PORT_USART1) || (portConfig->identifier == SERIAL_PORT_USART2) || (portConfig->identifier == SERIAL_PORT_USART3))
```

Comment:
better to check that it's NOT a softserial port, we support many more than 3 uarts now.

Reply:
the old `hottIsUsingHardwareUART` that this pr deleted did this.